src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        Color[] color = Color.values();
+        return color[random.nextInt(color.length)].name();
```

Comment:
Consider returning a `Color` type instead of a `String` from the `getRandomColor` method. This will ensure type consistency across the application when dealing with colors.